public/posts/deno-in-2020.md

```diff
@@ -0,0 +1,309 @@
+- [1.4.1](https://github.com/denoland/deno/releases/tag/v1.4.1)
+- [1.4.2](https://github.com/denoland/deno/releases/tag/v1.4.2)
+
+### October: REPL revamp, improved bundling, isolatedModules by default
+
+[1.5.0 blog post](https://deno.land/posts/v1.5)
+
+The biggest change that happened in this month was enabling `isolatedModules`
+option in TypeScript compiler host by default. This setting changes the behavior
+of TypeScript in such a way that ensures that each file can be transpiled in
+isolation (without knowledge of types and/or other modules) by tools other than
+TSC like SWC and Babel. This change had a significant impact on the module
+ecosystem, making some popular modules unusable until maintainers adjusted the
+code to work with `isolatedModules`.
+
+This month we also adopted the new bundle feature in SWC, yet another step in
+the direction of using Rust over the original TypeScript compiler.
+
+**Releases that month:**
+
+- [1.4.3](https://github.com/denoland/deno/releases/tag/v1.4.3)
+- [1.4.4](https://github.com/denoland/deno/releases/tag/v1.4.4)
+- [1.4.5](https://github.com/denoland/deno/releases/tag/v1.4.5)
+- [1.4.6](https://github.com/denoland/deno/releases/tag/v1.4.6)
+- [1.5.0](https://github.com/denoland/deno/releases/tag/v1.5.0)
+- [1.5.1](https://github.com/denoland/deno/releases/tag/v1.5.1)
+
+### November: Grand rewrite of TSC compiler infrastructure
+
+This month we saw a conclusion to [Kitson Kelly's](https://github.com/kitsonk)
+weeks-long project of rewrite compilation pipeline. It improved the speed of
+TypeScript transpilation even more, but most importantly paid off a lot of
+technical debt.
+
+The [deno_crypto op crate](https://crates.io/crates/deno_crypto) was added.
+
+**Releases that month:** 1.5.2, 1.5.3, 1.5.4
+
+### December: Self-contained binaries and LSP
+
+[1.6.0 blog post](https://deno.land/posts/v1.6)
+
+In December we released version 1.6 containing two milestone features:
+self-contained binaries and the language server. `deno compile` was single most
+requested feature in Deno’s bug tracker.
+
+Providing built-in language server allows to provide great development
+experience to all editors that can talk LSP protocol. It leads to third revamp
+of vscode_code that is still work-in-progress.
+
+**Releases that month:**
+
+- [1.6.0](https://github.com/denoland/deno/releases/tag/v1.6.0)
+- [1.6.1](https://github.com/denoland/deno/releases/tag/v1.6.1)
+- [1.6.2](https://github.com/denoland/deno/releases/tag/v1.6.2)
+- [1.6.3](https://github.com/denoland/deno/releases/tag/v1.6.3)
+
+### 2021
+
+We've seen a lot of growth in the project and community in 2020. Going into 2021
+we feel strongly about momentum behind Deno. Stay tuned for some exciting
+announcements coming soon!
+
+If you're interested in contributing to Deno or just want to follow our progress
+please look into the following:
+
+- Answer [the Deno survey](https://forms.gle/hbhP46LUAfVFMggU6).
```

Comment:
Bit of feedback on the survey: the question about using JavaScript and/or TypeScript should probably be a checkbox rather than a radio button

Reply:
ditto for the question about which parts of the std lib have you used

Reply:
... and the kind of software built with Deno

Reply:
Thanks, updated